app/controllers/application_controller.rb

```diff
@@ -20,4 +20,10 @@ def current_user
   def logged_in?
     current_user.persisted?
   end
+
+  def send_email(user:, contents:, link:)
+    EmailService.new(user: @user, 
+                      contents: contents,
+                      link: link)
+  end
```

Comment:
```suggestion
  def send_email(user: @user, contents:, link:)
    EmailService.new(user: user, contents: contents, link: link)
  end
```

I think you'd want that so your `user:` param is actually used.